Makefile

```diff
@@ -56,6 +56,16 @@ DEFINES += BLE_SEGMENT_SIZE=32
 DEFINES += HAVE_WEBUSB WEBUSB_URL_SIZE_B=0 WEBUSB_URL=""
 DEFINES += UNUSED\(x\)=\(void\)x
 
+ifeq ($(TARGET_NAME),TARGET_NANOS)
+    DEFINES += MAX_TRANSACTION_PACKETS=6
+endif
+ifeq ($(TARGET_NAME),TARGET_NANOS2)
+    DEFINES += MAX_TRANSACTION_PACKETS=106
```

Comment:
Yes, the last version had a limit of 510 bytes per transaction - enough to make native coin transfers, but not enough to call functions with many parameters. For example, it is still impossible to sign a [package publishing transaction](https://explorer.aptoslabs.com/txn/394773186/payload), since it is about 100Kb, but using the maximum available memory per device (~27Kb for LNS+) allows us to sign most of the typical transactions.

Reply:
For me, this approach is okay as is